atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -292,7 +292,7 @@ def set_group_manifest_info(self, extra, worker_metadatas):
         tags = [image.tag for image in primary_images]
 
         manifest_list_digests = self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY)
-        if manifest_list_digests:
+        if manifest_list_digests and len(manifest_list_digests) > 0:
```

Comment:
Checking length is redundant, isn't it? Empty dict evaluates to falsy.

Reply:
Agreed, fixed